### ISSUE TITLE
[Anvil App Server v1.6.5] anvil.users.force_login and anvil.users.login_with_email

### ISSUE THREAD

**Issue by JackN-code** [anvil-works/anvil-runtime#53]
The below code works find on the cloud Anvil but get error when using the Anvil App Server v1.6.5

```
@anvil.server.callable
def my_login(email, password):
  user = app_tables.users.get(email=email)  
  
  if user is not None:
    print (type(user))
    #anvil.users.force_login(user)
    #anvil.users.login_with_email(email=email, password = password,remember=True)
  else:
    return "Wrong info"

```

```

anvil.users.login_with_email(email=email, password = password,remember=True)
TableError: This table cannot be written or searched by this app

anvil.users.force_login(user)
InternalError: force_login() must be passed a row from the users table
```

**Comment by meredydd** [comment]
Hi @JackN-code,

Can you provide a replication case for this? (ie an app that displays the error, or a procedure for generating this error?)

**Comment by JackN-code** [comment]
Hi @meredydd, I clone the Customising the Users service app from official doc:
https://anvil.works/docs/how-to/custom-user-auth#customising-the-users-service
https://anvil.works/build#clone:3HBGAFAGNJ6FUQZV%3d4NPQMV4R47BAOIHSXTVRLP5R

```
client code
anvil.server.call('my_login',d.email_box.text, d.password_box.text)

@anvil.server.callable
def my_login(email, password):
  user = app_tables.users.get(email = email)
  if user is not None:
    user['confirmed_email'] = True 
    anvil.users.login_with_email(email, password, remember=True)
  else:
    return "wrong data"
  pass

```
Link the app in my server:
https://159.223.51.159/

**Comment by meredydd** [comment]
This should now be fixed in v1.7.0.
